Dictionary.java

```diff
@@ -0,0 +1,49 @@
+import java.util.LinkedList;
+
+public class Dictionary {
+	private LinkedList<String> words;

```

Comment:
Yes, HashSet is faster thank LinkedList. I have changed the data structures to that.